fvtest/porttest/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ if(OMR_OPT_CUDA)
 	)
 endif()
 
-if(OMR_PORT_SOCKET_SUPPORT)
+if((OMR_PORT_SOCKET_SUPPORT) AND (NOT OMR_HOST_OS STREQUAL "win"))
```

Comment:
It should have a TODO comment indicating to remove `AND (NOT OMR_HOST_OS STREQUAL "win")` from the boolean expression after the OMRSock API is implemented on Windows.